src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -998,66 +998,76 @@ static inline int MPIDI_NM_mpi_win_allocate_shared(MPI_Aint size,
         MPIR_ERR_SETANDSTMT(mpi_errno, MPI_ERR_NO_MEM, goto fn_fail, "**nomem");
     }
 
-    if (comm_ptr->rank == 0) {
-        int map_flags = MAP_SHARED;
+    int iter = MPIR_CVAR_CH4_SHM_SYMHEAP_RETRY;
+    unsigned anyfail = 1;
```

Comment:
Actually I personally preferred to declare variables at the beginning of every block. But it seems all ch4 files are changed to follow the C99 style which allows declaration in the middle of an executable statement. This somehow gives a little better code readability in my experience.